exercises/accepting_input/scaffold/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "author": "",
+  "description": "",
+  "keywords": [],
+  "license": "ISC",
+  "main": "index.js",
+  "name": "scaffold",
+  "version": "1.0.0",
+  "dependencies": {
+    "@hapi/iron": "^6.0.0",
+    "accepts": "^1.3.7",
+    "ajv": "^6.12.2",
+    "are-we-dev": "^1.0.0",
+    "bole": "^4.0.0",
+    "cookie": "^0.4.1",
+    "culture-ships": "^1.0.0",
+    "dotenv": "^8.2.0",
+    "find-my-way": "^3.0.4",
+    "uuid": "^8.3.0"
+  },
+  "devDependencies": {
+    "@hapi/shot": "^5.0.4",
+    "@typescript-eslint/eslint-plugin": "^4.11.0",
+    "@typescript-eslint/parser": "^4.11.0",
+    "ansi-escapes": "^4.3.1",
+    "bistre": "^1.0.1",
+    "eslint": "^7.1.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-prettier": "^3.1.3",
+    "get-function-location": "^2.0.0",
+    "nodemon": "^2.0.4",
+    "prettier": "^2.0.5",
+    "supports-hyperlinks": "^2.1.0",
+    "tap": "^14.10.7"
+  },
+  "scripts": {
+    "boltzmann:docs": "npx boltzmann-cli --docs",
+    "boltzmann:routes": "node -e 'require(\"./boltzmann\").printRoutes()'",
+    "boltzmann:upgrade": "npx boltzmann-cli",
+    "lint": "eslint .",
+    "posttest": "npm run lint",
+    "start": "nodemon ./boltzmann.js",
+    "test": "tap tests"
```

Comment:
(I note, I've started replacing the `test` script in our scaffolds with `boltzshopper run .`)